templates/summary-tpl.php

```diff
@@ -9,19 +9,16 @@
 
 ?>
 
-<?php
-require 'timeRange.include.php';
-?>
+<?php require_once 'timeRange.include.php'; ?>
+
+<?php require_once 'functions.include.php'; ?>
```

Comment:
Please remove the empty line and put the 'require_once' commands into one php code block